gstd/gstd_pipeline.c

```diff
@@ -109,10 +112,15 @@ struct _GstdPipeline
    */
   gint64 duration;
 
-    /**
+  /**
    * Pipeline graph with GraphViz dot format
    */
   gchar* graph;
+ 
+  /**
+   * Verbose state 
```

Comment:
The is not a helpful comment. Consider something that will orient other developer's more. Something like: `Enable/disable deep notify logging (similar to adding -v to get-launch-1.0)`